src/stories/components/404/NotFound404.tsx

```diff
@@ -1,36 +1,33 @@
-import styled from '@emotion/styled';
-import { Typography, useMediaQuery } from '@mui/material';
+import { Typography, styled, useMediaQuery, useTheme } from '@mui/material';
 import Image from 'next/image';
 import { useRouter } from 'next/router';
-import React from 'react';
 import Background404 from '../../../../public/assets/img/background-404.png';
 import Background404Dark from '../../../../public/assets/img/background-dark-404.png';
 import BackgroundMobile404 from '../../../../public/assets/img/background-mobile-404.png';
 import BackgroundMobile404Dark from '../../../../public/assets/img/background-mobile-dark-404.png';
 import Logo404 from '../../../../public/assets/img/logo-404.png';
 import Logo404Dark from '../../../../public/assets/img/logo-dark-404.png';
-import lightTheme from '../../../../styles/theme/themes';
-import { useThemeContext } from '../../../core/context/ThemeContext';
 import { CustomButton } from '../CustomButton/CustomButton';
+import type { Theme } from '@mui/material';
+import type { FC } from 'react';
 
-const NotFound404: React.FC = () => {
+const NotFound404: FC = () => {
   const router = useRouter();
-  const phoneLess = useMediaQuery(lightTheme.breakpoints.down('mobile_375'));
-  const isMobile = useMediaQuery(lightTheme.breakpoints.between('mobile_375', 'table_834'));
-  const { isLight } = useThemeContext();
+  const theme = useTheme();
+  const isMobile = useMediaQuery((theme: Theme) => theme.breakpoints.down('table_834'));
```

Comment:
Same than above, this is a legacy breakpoint

Reply:
Done